docs/user-guides/pallet-proxy.md

```diff
@@ -0,0 +1,191 @@
+---
+sidebar_position: 16
+---
+
+# Introduction to Pallet Proxy 
+
+**Why use Proxy?**
+---
+Pallet Proxy enables accounts to grant specific permissions to other accounts, empowering them to make calls on their behalf, thereby safeguarding the security of the underlying accounts.
```

Comment:
Can we add an introduction block of anonymous (= pure) proxy?
This is for sure more dangerous for non experimented users but it's also the best security since it's the only way to have a proxy that can't be controlled from private keys.
https://wiki.polkadot.network/docs/learn-proxies#anonymous-proxy-pure-proxy

Reply:
will do.